### ISSUE TITLE
Remote input doesn't work with Japanese

### ISSUE THREAD

**Issue by nicolasmaia** [ZorinOS/zorin-connect-android#5]
When using mozc on my phone, whenever I input text in Japanese, numbers are displayed instead of the desired characters.

This issue is not present when I type in English using the default IME.

**Comment by howdev** [comment]
yes I realised the same, non-English characters does not work. I don't see why this should not work. Other remote input apps sends whatever the language is.
